### PR TITLE
Use string.ascii_lowercase instead of str.lowercase

### PR DESCRIPTION
**Pull request by blueyed**
Fixes https://github.com/davidhalter/jedi-vim/issues/478.

### CODE REVIEW

File: jedi_vim.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def goto(mode="goto", no_output=False):
                         tagname = d.name
                         while vim_eval('taglist("^%s$")' % tagname) != []:
                             tagname = d.name + ' ' + ''.join(
-                                [random.choice(string.lowercase)
+                                [random.choice(string.lower)
```
Comment (davidhalter) [inline jedi_vim.py]: Why are we even joining the same thing four times? Besides: string.lower is a function and I don't even find a definition for `string.`

Comment (davidhalter) [inline jedi_vim.py]: Ok. I just realized `string` is a module. `string.lowercase` doesn't exist in Python 3 anymore. It's named `ascii_lowercase` there.

You can just defined it manually:

```
>>> string.lowercase
'abcdefghijklmnopqrstuvwxyz'
```

Comment (blueyed) [inline jedi_vim.py]: Yeah, the fix is not correct either.

`string` is imported as a module, and `string.lowercase` exists in Python 2: `'abcdefghijklmnopqrstuvwxyz'`.

`string.ascii_lowercase` should be used for both Python 2/3.

**Comment by blueyed** [comment]
Pushed an amended commit.

**Pull request closed without merging**